### ISSUE TITLE
Questions about code

### ISSUE THREAD

**Issue by super233** [junjzhang/MetaCropper#1]
Hi, thanks for your sharing awesome code.

Is the paper "Learning to Learn Cropping Models for Different Aspect Ratio Requirements" reproduced completely? I do not find the entrance to run your code. If it's possible, could you please share the full code?

**Comment by cip8** [comment]
Indeed, some extra details & a few examples on how this works would be highly appreciated - thank you! ❤️

**Comment by junjzhang** [comment]
Sorry it's been three years past before I saw the message. Not sure whether you guys still need my codes, but I'll fill in the code if I have time in the next couple weeks. BTW, although I'm not a fan of meta learning (till now), I remember this model's performance is decent.

**Comment by ppdk-data** [comment]
Great work!
I urgently need this project. It's very helpful if you can fill the code.
